pandas/_libs/tslibs/parsing.pyx

```diff
@@ -646,7 +648,10 @@ cdef datetime dateutil_parse(
         str reso = None
         dict repl = {}
 
-    res, _ = DEFAULTPARSER._parse(timestr, dayfirst=dayfirst, yearfirst=yearfirst)
+    try:
+        res, _ = DEFAULTPARSER._parse(timestr, dayfirst=dayfirst, yearfirst=yearfirst)
+    except InvalidOperation:
+        res = None
```

Comment:
is there a test which hits this? if not, could you add one please, else someone might remove it thinking it's not needed

Reply:
I will look into it. Thank you!

Reply:
I think something like `to_datetime(['87156549591102612381000001219H5'])` might be enough to hit it, could you put such an example in `test_to_datetime`?

Reply:
Updated as per the comments.